dbt_project.yml

```diff
@@ -160,10 +161,18 @@ on-run-end:
 models:
   snowplow_unified:
     +materialized: table
-    +file_format: delta
+    +file_format: >
+      {{ 
+        var(
+          'snowplow__datalake_file_format',
+          'delta' if target.type != 'spark' else 'iceberg'
+        ) 
+      }}
+    +incremental_strategy: "{{ 'delete+insert' if target.type in ['postgres', 'redshift'] else 'merge' }}"
```

Comment:
Same with the utils pr, flagging this not to forget as I'm not sure why this is needed yet.

Reply:
Addressed on the other pr